app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    assert 17 == sum(split_integer(17, 4))
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert [8, 8, 8, 8, 8] == split_integer(40, 5)
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert 1 == len(split_integer(25, 1))
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    assert [5, 5, 5, 5, 6, 6] == sorted(split_integer(32, 6))
```

Comment:
this test is incorrect. You need to test that the result is sorted by function `split_integer` but in your test, you receive the result of this function and then sort it so of course it will be sorted and you don't check anything